src/components/RegionSelect/Russia/RussianCitiesSelect.tsx

```diff
@@ -0,0 +1,181 @@
+        onChange={handleChange}
+        inputValue={cityInput}
+        variant="halfWidth"
+      />
+      <div className="location-selector__alphabet mt-12 flex justify-center w-full">
+        <div className="alphabet relative flex justify-center mr-14">
+          <noindex>
+            <ul className="alphabet__content flex justify-center w-[416px] flex-wrap">
+              {alphabet.map(({ active, inactive }) => {
+                return (
+                  <li className="font-rex text-orange-title text-base mr-1.5">
+                    <Link
+                      className={`${
+                        inactive
+                          ? 'text-neutral-300'
+                          : 'text-orange-title hover:underline hover:underline-offset-4 decoration-2'
+                      } alphabet__trigger inline-block pr-2
+                                  w-5
+                                  h-5
+                                  text-center
+                                  uppercase
+                                  border-1 border-transparent`}
+                      href="#letter"
+                    >
+                      {active} {inactive}
+                    </Link>
+                  </li>
+                );
+              })}
+            </ul>
+          </noindex>
+        </div>
+        <div className="location-selector__links flex self-center justify-between w-[290px]">
+          <Link
+            className="location-selector__top h-[71px] ease-in-out duration-200 bg-transparent"
+            href="/"
+          >
+            <Image
+              src="/images/logo/location-moscow.png"
+              alt="Oldboy Barbershop Moscow logo"
+              className="h-full 
+                          hover:transition-filter 
+                          hover:duration-200 
+                          hover:ease 
+                          hover:opacity-100 
+                          hover:drop-shadow-[1px_1px_0_orange]"
+              width={126}
+              height={100}
+              onClick={() => handlePickCity('Москва')}
+            />
+          </Link>
+          <Link
+            className="location-selector__top h-[71px] ease-in-out duration-200 bg-transparent"
+            href="/"
+          >
+            <Image
+              src="/images/logo/location-peterburg.png"
+              alt="Oldboy Barbershop St. Petersburg logo"
+              className="h-full
+                            hover:transition-filter 
+                            hover:duration-200 
+                            hover:ease 
+                            hover:opacity-100 
+                            hover:drop-shadow-[1px_1px_0_orange]"
+              width={126}
+              height={100}
+              onClick={() => handlePickCity('Санкт-Петербург')}
+            />
+          </Link>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const RussianLocationScroll = (): ReactElement => {
```

Comment:
only 1 component per file, this should be moved to a separate folder (don;'t forget index.ts)